doc/dev/decisionrecords/NamingConventions.md

```diff
@@ -1,74 +1,71 @@
+
+| Good method prefixes                                  | Description                                                                   |
+|-------------------------------------------------------|-------------------------------------------------------------------------------|
+| `stop() : Stop`                                       | Field accessor, equivalent to `getStop` as in the Java Bean standard.         |
+| `getStop(ID id) : Stop`                               | Get Stop by ID; throws exception if not found.                                |
+| `getStops(Collection<ID> id) : List/Collection<Stop>` | Get _all_ Stops by set of IDs; throws exception if not found.                 |
+| `findStop(Criteria criteria) : Optional<Stop>`        | Find one or zero stops; return `Optional`.                                    |
+| `findStops(Criteria criteria) : List/Stream<Stop>`    | Find 0, 1, or many stops; return a Collection or Stream (List is preferred).  |
+| `listStops() : List/Stream<Stop>`                     | List ALL stops in context; return a Collection or Stream (List is preferred). |
+| `withStop(Stop stop) : Builder`                       | Set Stop in builder, replacing existing value; return `this` builder.         |
+| `initStop(Stop stop) : void`                          | Set property _once_; a second call throws an exception.                       |
+| `addStop(Stop stop) : void/Builder`                   | Add a Stop to a collection of Stops.                                          |
+| `addStops(Collection<Stop> stops) : void/Builder`     | Add set of Stops to existing set.                                             |
+| `withBike(Consumer<BikePref.Builder> body) : Builder` | For nested builders, use lambdas.                                             |
+
+These prefixes are also "allowed" but not preferred; they have some kind of negative "force" to
+them.
+
+| Okay method prefixes, but ...               | Description                                                                                            |
+|---------------------------------------------|--------------------------------------------------------------------------------------------------------|
+| `withStops(Collection<Stop> stops) : this`) | Replace all Stops in builder with new set. Consider using `addStops(...)` instead.                     |
+| `setStop(Stop stop)`                        | Set a mutable Stop reference. Avoid if not part of natural lifecycle. Use `initStop(...)` if possible. |
+| `getStop() : Stop`                          | Old style accessor. Use the shorter form `stop() : Stop`.                                              |
 
 ## Service, Model and Repository
 
 ![MainModelOverview](../images/ServiceModelOverview.png)
 
-
-
 Naming convention for builders with and without a context.
 
-#### Graph Build and tests run without a context
+#### Graph builds and tests run without a context
```

Comment:
I think we probably should stick to title case in all subtitles as well.

Reply:
Done.